docs/guides/package-lock.json

```diff
@@ -15,7 +15,7 @@
       "integrity": "sha512-VT0ZI6kZRdTh8YyJw3SMbYm/u+NqfsAxEpWO0Pf9sq8/e94WxxOpPKx9FR1FlyCtOVDNOQ+8ntlqFxiRc+r5qA==",
```

Comment:
I feel like this file wasn't meant to be updated.  What did you change here, and was it deliberate?

Reply:
This file wasn't meant to be updated. I used the npm install to get into node_modules to use the style check for the markdown file. Could the changes come from there?

Reply:
Yep.  I would rebase this change out of the history since it's irrelevant.